tensorboard/webapp/tb_polymer_interop_type_definitions.d.ts

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+declare global {
+  interface Window {
+    tensorboard: {
+      tf_storage: Partial<TfStorage>;
```

Comment:
Can we remove Partial from this? Maybe this is enabled by some of the other comments I make.